src/cx/cx.ts

```diff
@@ -116,7 +116,11 @@ export class Cx implements CxPlatform {
 		params.set(CxParamType.AGENT, constants.vsCodeAgent);
 		params.set(CxParamType.ADDITIONAL_PARAMETERS, constants.scanCreateAdditionalParameters);
 		const scan = await cx.scanCreate(params);
-		return scan.payload[0];
+		
+		if (scan.payload && scan.exitCode===0) {
+			return scan.payload[0];
+		}
+		throw new Error(scan.status);
```

Comment:
Should we throw Error object? I saw in vscode-extension that we have ASTError class that extends Error